iceoryx_hoofs/include/iceoryx_hoofs/internal/posix_wrapper/shared_memory_object.hpp

```diff
@@ -19,7 +19,7 @@
 
 #include "iceoryx_hoofs/cxx/filesystem.hpp"
 #include "iceoryx_hoofs/cxx/optional.hpp"
-#include "iceoryx_hoofs/design_pattern/creation.hpp"
+#include "iceoryx_hoofs/design_pattern/builder_pattern.hpp"
```

Comment:
Assuming this is kept in `design_pattern`, why repeating `pattern` and not just `builder.hpp`

Reply:
When we decide to leave this in `design_pattern` I will remove the `pattern` from the builder.